edgexfoundry/app-functions-sdk-go
# feat: Consume watch for common Writable config changes

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

      <link to docs PR>

## Testing Instructions
Run EdgeX non-secure stack 
Build app template from this branch and run from command line with `-cp`
Verify logs contain the following:
```
"listening for private config changes"
"listening for all services common config changes"
"listening for application service common config changes" 
```
In Consul add `Writable/Telemetry/Interval` to `edgex/v3/core-common-config-bootstrapper/app-services` with value of 20s
Verify the log contains:
```
"Writeable configuration has been updated from the Configuration Provider"
"Telemetry interval has been updated. Processing new value..."
"Metrics Manager report interval changed to 20s"
```
## New Dependency Instructions (If applicable)
